framework/helpers/Url.php

```diff
@@ -15,4 +15,31 @@
  */
 class Url extends BaseUrl
 {
+    /**
+     * @param        $params  Array of values to convert into http query
+     * @param string $keyfather Inheritance of keys
+     * @param string $response
+     * @param bool   $ltrim
+     *
+     * @return string
+     */
+    public static function custom_http_build_query($params, $keyfather = '', $response = '', $ltrim = true) {
```

Comment:
I wrote this myself. I'll describe the arguments more, if thought that they are mostly self explainable by the name but more explanation wouldn't hurt anybody. 
Yii is only using array values as parameteres when calling http_build_query. If needed I'll implement the object aswell but I'm not really sure is needed.